src/stan-users-guide/proportionality-constants.Rmd

```diff
@@ -0,0 +1,150 @@
+# Proportionality Constants  {#proportionality-constants.chapter}
+
+When evaluating a likelihood or prior as part of the log density computation in
+MCMC, variational inference, or optimization, it is usually only necessary to
+compute the functions up to a proportionality constant (or similarly compute
+log densities up to an additive constant). In MCMC this comes from the fact that
```

Comment:
In relation to the question you asked about which terminology to use, I think you've done a good job here! You use both "proportionality constant" and "additive constant" but you make it clear that additive is when working on the log scale.